bin/src/Main.purs

```diff
@@ -819,10 +819,9 @@ mkReplEnv replArgs dependencies = do
 
   purs <- Purs.getPurs
 
-  let
-    selected = case workspace.selected of
-      Just s -> [ s ]
-      Nothing -> Config.getWorkspacePackages workspace.packageSet
+  selected <- case workspace.selected of
+    Just s -> pure s
+    Nothing -> die "The REPL requires you to select a package"
```

Comment:
Here's one potential downside of this change... But I guess this is just making what was an implicit problem explicit...?

Reply:
Do we have to change this?

Reply:
What happens when there are two modules with the same package name? I guess the only option would be to force the user to specify a package that exists only for the repl?

Reply:
You mean two packages with the same module name? We shouldn't allow that - we'll probably need to run a graph check when compiling more than one package so we can detect this.

Reply:
I've updated the code to reallow this, but haven't yet implemented the duplicate module check yet.